cfgov/unprocessed/js/molecules/Notification.js

```diff
@@ -133,6 +133,10 @@ function Notification( element ) { // eslint-disable-line max-statements, inline
     return this;
   }
 
+  function getElement() {
```

Comment:
I'm using this for on success form replacement for forms with backgrounds. Error notifications should show inside the background:

<img width="804" alt="screen shot 2017-07-18 at 10 59 20 am" src="https://user-images.githubusercontent.com/778171/28324109-a6f7951a-6ba8-11e7-9730-82ddcc4a79fb.png">
 
but the background isn't necessary when the form gets replaced with the success message:
<img width="817" alt="screen shot 2017-07-18 at 11 15 26 am" src="https://user-images.githubusercontent.com/778171/28324920-8ee69aaa-6baa-11e7-8e5f-0935b508d21d.png">

so in the FormSubmit form replacement code, I swap out the form for the notification element it contains.